File: module_me/src/main/java/debug/MainActivity.java
package debug;


import androidx.fragment.app.Fragment;
import androidx.fragment.app.FragmentTransaction;

import com.alibaba.android.arouter.facade.annotation.Autowired;
import com.ebook.common.mvvm.BaseActivity;
import com.ebook.common.provider.IMeProvider;
import com.ebook.me.R;

public class MainActivity extends BaseActivity {
    @Autowired(name = "/me/main")
    IMeProvider mMeProvider;

    private Fragment mMeFragment;

    @Override
    public int onBindLayout() {
        return R.layout.activity_main;
    }

    @Override
    public void initView() {
        if (mMeProvider != null) {
            mMeFragment = mMeProvider.getMainMeFragment();
        }
        FragmentTransaction transaction = getSupportFragmentManager().beginTransaction();
        if (!mMeFragment.isAdded()) {
            transaction.add(R.id.frame_content, mMeFragment, "ME").commit();
        } else {
            transaction.show(mMeFragment).commit();
        }
    }

    @Override
    public void initData() {

    }
}

File: module_me/src/main/java/debug/MainActivity.kt
package debug


import android.view.LayoutInflater
import android.view.ViewGroup
import com.ebook.common.provider.IMeProvider
import com.ebook.me.R
import com.ebook.me.databinding.ActivityMainBinding
import com.therouter.TheRouter
import com.xrn1997.common.mvvm.view.BaseActivity

class MainActivity : BaseActivity<ActivityMainBinding>() {
    override fun initView() {
        val mMeFragment = TheRouter.get(IMeProvider::class.java)?.getMainMeFragment()
        val transaction = supportFragmentManager.beginTransaction()
        if (mMeFragment != null) {
            if (!mMeFragment.isAdded) {
                transaction.add(R.id.frame_content, mMeFragment, "ME")
            } else {
                transaction.show(mMeFragment)
            }
        }
        transaction.commit()
    }

    override fun initData() {}
    override fun onBindViewBinding(
        inflater: LayoutInflater,
        container: ViewGroup?,
        attachToParent: Boolean
    ): ActivityMainBinding {
        return ActivityMainBinding.inflate(inflater, container, attachToParent)
    }
}
